### DISCUSSION TITLE
How to download latest beta or compose apk

### DISCUSSION THREAD

**Discussion by jxparrow** [z-huang/music#638] [Q&A] [answered]
I just wanted to try out new features please somone guide me i am noob at this i wanted to try dynamic theme and i couldn't figure out the way to download latest beta or compose apk or anything ao that i can install apk and use it please help me

**Comment by nikhilCad** [comment, marked as answer]
https://github.com/z-huang/InnerTune/actions

Download 9nes with green check mark. App will be slow as it's debug build. Need to sign in to Github
